scripts/generate-plugins.go

```diff
@@ -136,15 +136,17 @@ func listDirectories(path string) ([]string, error) {
 
 	for _, item := range items {
 		// We only want directories
-		if item.IsDir() {
-			currentDir := filepath.Join(path, item.Name())
-			names = append(names, currentDir)
-
-			// Do some recursion
-			subNames, err := listDirectories(currentDir)
-			if err == nil {
-				names = append(names, subNames...)
-			}
+		if !item.IsDir() ||
+			item.Name() == "common" {
+			continue
+		}
+		currentDir := filepath.Join(path, item.Name())
+		names = append(names, currentDir)
+
+		// Do some recursion
+		subNames, err := listDirectories(currentDir)
+		if err == nil {
+			names = append(names, subNames...)
```

Comment:
This code generator here lists all files in the `builder` dir and see if any contain a `Builder` struct: so because this PR moved the `proxmox.Builder` to `common.Builder` this generator was creating the `proxmox-common` builder.
I think this is okay to have a `common.Builder` in terms of naming so I made it so that the script ignores "common" packages.

Now because we now have `proxmox/iso` and `proxmox/clone`, the `proxmox` builder is gone, make of this PR currently a breaking change.
One thing we could do is create a dummy Package in `proxmox` that contains a type alias :
```go
package proxmox

type Builder  = common.Builder
```

In order to make this a non-breaking-change, I would also add a [fixer](https://github.com/hashicorp/packer/tree/master/fix) struct that will transform all proxmox builders into their counterpart.

Reply:
Oh that works - I'll head in that direction to allow for compatibility. Not familiar with `fixer` structs, but I'll check it out.